src/domain/repositories/UserRepository.ts

```diff
@@ -3,5 +3,5 @@ import { Id } from "$/domain/entities/Ref";
 import { User } from "$/domain/entities/User";
 
 export interface UserRepository {
-    getCurrent(authorizationGroups: { adminGroups: Id[] }): FutureData<User>;
+    getCurrent(options: { adminGroups: Id[] }): FutureData<User>;
```

Comment:
If you wanted to make the option optional: `getCurrent(options: { adminGroups?: Id[] })`